pkg/services/authn/authnimpl/service.go

```diff
@@ -152,6 +152,10 @@ func ProvideService(
 		s.RegisterPostAuthHook(sync.ProvideOAuthTokenSync(oauthTokenService, sessionService).SyncOauthTokenHook, 60)
 	}
 
+	if features.IsEnabled(featuremgmt.FlagAuthnService) {
```

Comment:
We don't need to check the flag here. The service are not used if this flag is not set